_posts/2020-02-06-Block-Device-Compatibility.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Block Device Compatibility
+image: /img/posts/iconfinder_Computer__Computer_Hardware_SSD_Storage_Technology_4064142.png
+categories: [pmem, SSD, compatibility]
+---
+
+The block device compatibility layer of *go-pmem* was developed so that
+our package can be used even on those machines without persistent memory (pmem).
+
+This layer provides weaker consistency guarantee than *go-pmem* running on
+persistent memory, but lets developers use and test our package on machines
+which has only block I/O storage. The weaker consistency guarantees offered by
+this compatibility layer are the following:
+* Our package is resilient to application crashes or restarts
+* In case of a host crash or restart, data can get irrecoverably corrupted
```

Comment:
rewrite as:

...are the following:
 * it is resilient to application crashes or restarts
 * ...